### ISSUE TITLE
Running Alto in Prod

### ISSUE THREAD

**Issue by Brianspha** [pimlicolabs/alto#215]
Hello,

I would like to ask if it's advisable to run Alto in production using the unsafe mode, I'm asking because we have been trying to get Alto to work by sending mint operation using the lightAccount but the bundler seems to revert it only works in unsafe mode is there something I'm doing wrong?

e.g. callData
```
  const callData = await lightAccount.encodeCallData({
    to: deployments.TestToken,
    data: encodeFunctionData({
      abi: ERC20_ABI,
      args: [
        lightAccount.address,
        new BigNumber(2).multipliedBy(10 ** 18).toFixed(0),
      ],
      functionName: "mint",
    }),
    value: 0n,
  });
```
Then we use a paymaster 
the **estimateUserOperationGas** endpoint manages to estimate gas but when we send the user operation we get issues
the bundler errors out with a response 

`{"message":"Cannot read properties of undefined (reading 'slice')"}`

**Comment by kristofgazso** [comment]
hey! thanks for this — do you know where that error gets thrown?

**Comment by Brianspha** [comment]
Hey @kristofgazso right here:
https://github.com/pimlicolabs/alto/blob/main/src/rpc/validation/SafeValidator.ts#L391

**Comment by Brianspha** [comment]
@kristofgazso  any updates on this?

**Comment by mouseless0x** [comment]
Hey, there was a issue with our SafeMode for a short period of time but this was addressed in #248 (this patch was included in docker version [v1.2.0](https://github.com/orgs/pimlicolabs/packages/container/alto/228630558?tag=v1.2.0))
